cirq/ops/moment.py

```diff
@@ -133,6 +133,16 @@ def operation_at(self, qubit: raw_types.Qid) -> Optional['cirq.Operation']:
             return self.__getitem__(qubit)
         else:
             return None
+   
+    def single_gate_in_moment(self) -> Optional['cirq.Gate']:
+        target = None
+        if self.operations:
```

Comment:
I think you meant `if not self.operations`, right?

Reply:
Fixed those issues; thanks for pointing it out!